### PR TITLE
Levers in Oldton Movalpolos will now raise the bridge to Newton Movalpolos

### PR DESCRIPTION
**Pull request by CriticalXI**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

- Implements the three levers in Oldton Movalpolos to raise the bridge to enter Newton Movalpolos.

### CODE REVIEW

File: sql/npc_list.sql [hunk 1]
```diff
@@ -734,11 +734,13 @@ INSERT INTO `npc_list` VALUES (16822477,'_0b0','Rusty Lever',0,173.021,5.666,-46
...
-
+INSERT INTO `npc_list` VALUES (16822481,'_0b6','NPC',0,175.000,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822482,'_0b4','DROPFLOOR_B',0,181.003,12.000,-60.009,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16822483,'_0b7','NPC',0,181.003,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822484,'_0b5','DROPFLOOR_C',0,187.003,12.000,-60.009,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16822485,'_0b8','NPC',0,187.003,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16822486,'_0b9','     ',0,181.003,16.299,-66.980,1,40,40,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16822487,'_0ba','     ',0,175.000,16.299,-66.980,1,40,40,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
```
Comment (TeoTwawki) [inline sql/npc_list.sql:743]: could you tag these that we couldn't find them in an existing cap? thanks for PR'ing!

File: scripts/zones/Oldton_Movalpolos/npcs/_0b1.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+-----------------------------------
+local entity = {}
+
+entity.onTrigger = function(player, npc)
+    local gate = GetNPCByID(npc:getID() + 4)
+    npc:openDoor(30)
+    gate:closeDoor(30)
+end
+
+return entity
```
Comment (Xaver-DaRed) [inline scripts/zones/Oldton_Movalpolos/npcs/_0b1.lua]: New line at the end here.

File: scripts/zones/Oldton_Movalpolos/npcs/_0b2.lua [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+-----------------------------------
+local entity = {}
+
+entity.onTrigger = function(player, npc)
+    local gate = GetNPCByID(npc:getID() + 5)
+    npc:openDoor(30)
+    gate:closeDoor(30)
+end
+
+return entity
```
Comment (Xaver-DaRed) [inline scripts/zones/Oldton_Movalpolos/npcs/_0b2.lua]: And here. Rest looks good^^

File: sql/npc_list.sql (hunk 1 above)
Comment (CriticalXI) [reply sql/npc_list.sql:743]: Got that noted and also added you as a co-author on the commit since you helped me discover most of this!

Comment (TeoTwawki) [reply sql/npc_list.sql:743]: well thank you <3

**Review by TeoTwawki — APPROVED** [review]

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**